libretroshare/src/gxs/rsgds.h

```diff
@@ -66,8 +66,8 @@ typedef std::map<RsGxsGroupId,RsGxsGrpMetaData*> RsGxsGrpMetaTemporaryMap;
 class GrpLocMetaData {
 
 public:
-    GrpLocMetaData(const GrpLocMetaData& meta){ grpId = meta.grpId; val = meta.val;}
-    GrpLocMetaData(){}
+    GrpLocMetaData(const GrpLocMetaData& meta): grpId(meta.grpId), val(meta.val) {}
+    GrpLocMetaData() = default;

```

Comment:
Now that you touch it, you should made this class a struct, remove public keyword, and this line too (the class have no "special" members like pointers so the default constructor would be generated anyway AFAIR )

Reply:
Done